hexrd/ui/calibration/pick_based_calibration.py

```diff
@@ -93,7 +102,8 @@ def instr(self):
 
     @property
     def plane_data(self):
-        self._plane_data.wavelength = self._instr.beam_energy
+        self._plane_data.wavelength = self.energy_cutoffs[-1]
+        self._plane_data.exclusions = None

```

Comment:
Do we need to make a deep copy of the plane data so the hexrdgui one doesn't get modified? (which will make the GUI out of sync with the underlying plane data)